markdown/pxf/HDFSWritablePXF.html.md.erb

```diff
@@ -136,7 +136,7 @@ Perform the following operations to use the PXF `HdfsTextSimple` profile to crea
 			    FORMAT 'CSV';
     ```
 
-    The table is created with the `'CSV'` `FORMAT` because the delimiter character used when creating the writable table was a comma `,`.
+    The table is created with the `'CSV'` `FORMAT` because the delimiter character with which you created the writable table was a comma `,`.
```

Comment:
This sounds a little awkward.  Edit to:

"Specify the the `'CSV'` `FORMAT` for the readable table, because you created the writable table with a comma ',' as the delimiter character.